compat-data.schema.json

```diff
@@ -109,15 +109,15 @@
         "__compat": { "$ref": "#/definitions/subfeature_set" }
       },
       "patternProperties":{
-        "^(?!__compat).*$" : { "$ref": "#/definitions/identifier" }
+        "^(?!__compat|.*\\.).*$" : { "$ref": "#/definitions/identifier" }
```

Comment:
For my education: in "\\.", the first "\" is escaping the second "\" (for the JSON), and the second "\" is escaping the "." (for the regexp). Is that right?